cmd/server/assets/admin/events/index.html

```diff
@@ -58,6 +58,10 @@ <h1>System event log</h1>
               </button>
             </div>
           </div>
+          <div class="form-check">
```

Comment:
I think this should be a hidden URL param - no need to edit the form.

Reply:
done... but that does seem a little hidden

Reply:
left the hidden input in always, so it shows up in the URL on search. At least that's find-able if a bit obscure